src/codeu/chat/common/ChangeAccessRequest.java

```diff
@@ -0,0 +1,64 @@
+package codeu.chat.common;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+import codeu.chat.util.Serializers;
+import codeu.chat.util.Serializer;
+import codeu.chat.util.Uuid;
+import codeu.chat.common.Access;
+
+public class ChangeAccessRequest implements Writeable {
+
+  // get the type of this Writeable as a String
+  @Override
+  public String getType() {
+    return CHANGE_ACCESS_REQUEST_STR;
+  }
+
+  // write this Writeable as a User
+  @Override
+  public void write(OutputStream out, Object value) throws IOException {
+    SERIALIZER.write(out, (ChangeAccessRequest)value);
+  }
+
+  // map from conversation id to Access
+  public final Uuid user;
+  public final Uuid conversation;
+  public final Access access;
+
+  public ChangeAccessRequest(Uuid user, Uuid conversation, Access access) {
```

Comment:
nit: prefer to keep instance variables and constructor at the top of the class (in that order).